### ISSUE TITLE
adding lemmas and dependency label options in the "rewriting" feature

### ISSUE THREAD

**Issue by mmahamdi** [Arborator/arborator-frontend#135]
In the annotation interface, it would be nice to extend the "rewriting graph" feature to all of the grew commands. 
Right now, only the "rewriting POS" option is available, and adding other types of commands (i.e N.lemma , E.label ...) seems to cause errors. 

Thanks !

**Comment by bguil** [comment]
I've tested a command which modifies a lemma and it seems to work fine.

The rule I have tested is: 
```rule r1 { pattern { N [upos=VERB, lemma<>"TEST"] } commands { N.lemma = "TEST" } }```

Note that the rule must be written to avoid non-termination problem. For instance, the rule 
```rule r1 { pattern { N [upos=VERB] } commands { N.lemma = "TEST" } }```
cause an error (which is not nicely reported to the user...)

Another way to avoid looping rule is to write a without clause. For the rule above, it would be:
```
rule r1 { 
  pattern { N [upos=VERB] } 
  without { N [lemma = "TEST"] }
  commands { N.lemma = "TEST" } 
}
```
See https://grew.fr/tutorial/04_termination/ for more details.

Can you give the Grew code of the rules you have tried to apply?

**Comment by mmahamdi** [comment]
I get it now, thanks! I added the restrictions and now it works. 
My rule was very similar to your second example :
	```rule 1 {pattern { N[ lemma="***_" ] } commands {N.upos="X"}}```

which usually works on line command and Grew-web (I just tried it on the latter).

> Note that the rule must be written to avoid non-termination problem. For instance, the rule
rule r1 { pattern { N [upos=VERB] } commands { N.lemma = "TEST" } }
cause an error (which is not nicely reported to the user...) 

How about giving the link to the tutorial when reporting the error ?
